vendor/cookbooks/rails/libraries/default.rb

```diff
@@ -16,5 +16,45 @@ def nginx_custom_configuration(app_info)
 
       empty_conf.merge(app_info["nginx_custom"] || {})
     end
+
+    # Returns a server path to certificate file
+    #
+    #   applications_root = '/u/apps/'
+    #   name = 'my_app'
+    #   app_info['ssl_certificate'] = 'my_cert.crt'
+    #   ssl_certificate(applications_root, name, app_info) # => /u/apps/my_app/shared/config/my_cert.crt'
+    #
+    #   applications_root = '/u/apps/'
+    #   name = 'my_app'
+    #   app_info['ssl_certificate'] = nil
+    #   ssl_certificate(applications_root, name, app_info) # => /u/apps/my_app/shared/config/my_app.crt'
+    #
+    def ssl_certificate(applications_root, name, app_info)
+      raise "Invalid application config given, no `ssl_info` present" unless ssl_info?(app_info)
+      
+      Pathname.new(applications_root).join(name, "shared", "config",
+                                           app_info["ssl_info"]["certificate"] || "#{name}.crt")
+    end
+
+    # See #ssl_certificate
+    #
+    def ssl_certificate_key(applications_root, name, app_info)
+      raise "Invalid application config given, no `ssl_info` present" unless ssl_info?(app_info)
+
+      Pathname.new(applications_root).join(name, "shared", "config",
+                                           app_info["ssl_info"]["certificate_key"] || "#{name}.key")
+    end
+
+    # Check if the app config has ssl_info section
+    #
+    def ssl_info?(app_info)
+      app_info.key?('ssl_info')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.